python/api/client.py

```
@@ -0,0 +1,520 @@
+import random
+import time
+import urllib.parse
+
+import httpx
+
+from .credential import Credential
+from .exceptions import ArgsException, NetworkException, ResponseCodeException
+
+__all__ = [
+    "Api",
+    "BiliClient",
```

Ruff (0.16.3)

[error] Undefined name `BiliClient` in `__all__`

(F822)